content/operate/rc/databases/create-database/create-essentials-database.md

```diff
@@ -32,6 +32,10 @@ This guide shows how to create an Essentials database.
 
 1. Choose a **Cloud Provider** and a **Region**.
 
+    A preview of Redis 7.4 is available for databases in selected regions. Select the **Redis 7.4 preview** toggle to view the regions where you can try Redis 7.4.
+
+    {{<image filename="images/rc/essentials-74-preview.png" alt="Turn on the Redis 7.4 preview toggle.">}}
```

Comment:
This image isn't displaying. It's not part of this PR, so maybe it still needs to be merged?

Reply:
Oops. I took it and then forgot to put it in the folder. Thanks for letting me know.